EPs/sep_data_spectrum.md

```diff
@@ -0,0 +1,108 @@
+# Add functionality for spectrum data
+
+
+author: Kyle Boone <!-- your name -->
+
+date-created: 2020 May 18 <!-- replace with the date you submit the APE -->
+
+date-last-revised: 2020 May 18 <!-- keep this up to date anytime something changes -->
+
+date-accepted: <!-- replace with accepted date -->
+
+status: Discussion  <!-- one of: Discussion, Accepted, Rejected -->
+
+
+## Abstract
+
+The goal of this proposal is to add functionality to sncosmo to handle spectrum data. We
+propose to add a SpectrumData class that represents a spectra taken of a supernova, and
+implement functionality to work with this data.
+
+
+## Detailed description
+
+Sncosmo is currently limited to working with photometric data. However, spectra are
+often obtained of supernovae, and it would be very useful if sncosmo could use this
+additional data. We propose to add functionality to sncosmo to work with spectra.
+
+There are several ways in which spectra can be used for supernova cosmology. First,
+spectra can provide additional constraints when fitting models to observations of a
+supernova. Models such as the SNEMO model that was recently added to sncosmo are
+designed to be able to fit individual spectra of each supernova rather than being
+limited to photometry. Surveys such as the SNfactory are obtaining spectrophotometric
+timeseries of supernovae that can then be fit directly with these models, potentially
+providing additional information beyond what is captured by photometry alone. 
+
+Given a spectrophotometric timeseries, it is possible to generate synthetic photometry
+in arbitrary bands. This photometry has many uses, such as comparing between experiments
+or using light curve fitters that take photometry as input. Sncosmo already provides
+functionality to generate photometry from spectral time series models, and this same
+functionality could be applied to spectrophotometric spectra.
+
+Less-well-calibrated spectra can also be useful for typing supernovae or constraining
+models. However, they often require some warping of the spectrum or removal of host
+galaxy contamination. This functionality is beyond the scope of this EP, but we would
+like to set up a framework to be able to include it in the future.
+
+
+## Branches and pull requests
+
+An initial discussion of this work can be found in sncosmo/sncosmo#262.
```

Comment:
Github seems to not render these issue links automatically in markdown documents, so it might be nice to insert explicit links.

Reply:
Interesting. They work in comments like this (e.g. sncosmo/sncosmo#262) but not in .md files. I'll update the URLs.